docs/contributors/code/testing-overview.md

```diff
@@ -274,23 +274,24 @@ You should never create or modify a snapshot directly, they are generated and up
 Snapshot are mostly targeted at component testing. They make us conscious of changes to a component's structure which makes them _ideal_ for refactoring. If a snapshot is kept up to date over the course of a series of commits, the snapshot diffs record the evolution of a component's structure. Pretty cool 😎
 
 ```js
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 import SolarSystem from 'solar-system';
-import { Mars } from 'planets';
 
 describe( 'SolarSystem', () => {
 	test( 'should render', () => {
-		const wrapper = shallow( <SolarSystem /> );
+		const { container } = render( <SolarSystem /> );
 
-		expect( wrapper ).toMatchSnapshot();
+		expect( container.firstChild ).toMatchSnapshot();
 	} );
 
 	test( 'should contain mars if planets is true', () => {
-		const wrapper = shallow( <SolarSystem planets /> );
+		const { container, getByText } = render( <SolarSystem planets /> );
```

Comment:
Let's recommend `screen` instead:
```jsx
import { render, screen } from '@testing-library/react';

const { container } = render( <SolarSystem planets /> );

expect( screen.getByText( /mars/i ) ).toBeInTheDocument();
```

More info: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
Good idea, we use it in a few places already. I read the same blog post while doing some refactor a while ago 👍🏻

Reply:
Yeah, absolutely — I actually remember reading that post too. Thank you for the suggestion, diego!

Reply:
Done in [`96aa0c7` (#34423)](https://github.com/WordPress/gutenberg/pull/34423/commits/96aa0c72458d286589c07015ac3742c35a968858)